module/Photo/Module.php

```diff
@@ -95,6 +95,26 @@ public function getServiceConfig()
                         $sm->get('photo_doctrine_em')
                     );
                 },
+                'photo_acl' => function ($sm) {
+                    $acl = $sm->get('acl');
+
+                    // add resources for this module
+                    $acl->addResource('photo');
+                    $acl->addResource('album');
+                    $acl->addResource('tag');
+
+                    // Guests are allowed to view photos and albums
+                    $acl->allow('guest', 'photo', 'view');
+                    $acl->allow('guest', 'album', 'view');
+
+                    // Users are allowed to view, remove and add tags
+                    $acl->allow('user', 'tag', array('view', 'add', 'remove'));
+
+                    // Users are allowed to download photos
+                    $acl->allow('user', 'photo', array('download', 'view_metadata'));
+
+                    return $acl;
+                },
```

Comment:
This one is based on the same branch as PR #162 which is why all the changes of PR #162 also get included here. This one probably should wait until PR #162 is resolved.